src/shared/components/query/CancerStudySelector.tsx

```diff
@@ -225,9 +239,12 @@ export default class CancerStudySelector extends React.Component<ICancerStudySel
                             <Observer>
                                 {() => {
 
-                                    const studyLimitReached = (this.store.selectableSelectedStudyIds.length > 50);
+                                    const studyLimitReached = (!this.checkReferenceGenome) ||
```

Comment:
seems odd to combine these two invalid "states" no?   I think we should have a separate message for studies from different reference genome

Reply:
agree. Let's have  a new message.

Reply:
added a new message for multiple reference genomes.